bench/sorting/Main.hs

```diff
@@ -49,38 +49,44 @@ benchNub :: (forall (s :: S). Term s PInteger) -> Int -> TestTree
 benchNub plen len =
     singleTest
         ("pnubSort, length " <> show len)
-        (ScriptBench $ pnubSort @PInteger @PList #$ preplicate # plen # 1)
+        (ScriptBench (pnubSort @PInteger @PList, preplicate # plen # 1))
```

Comment:
type applications with tick mark should be put into a parenthesis because of fourmolu bug.. 

like this
```hs
@( 'PList)
```

I'm not sure if we are enforcing ticked mark on LPE..

Reply:
I don't think type applications normally _get_ ticks. At the very least, I haven't been using them.